logger-kafka-consumer/src/test/java/com/trihydro/loggerkafkaconsumer/app/dataConverters/TimDataConverterTest.java

```diff
@@ -208,19 +208,6 @@ public void processTimJson_odeTimWithRsus() throws IOException {
 
         }
 
-        @Test
-        public void processTimJson_unsigned() throws IOException {
```

Comment:
Note (context): I believe this test case started failing after making some changes but since we don't intend to handle unsigned TIMs in CDOT the decision was made to just delete it.